CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Changelog
```

Comment:
We don't have one already 😱. Do you think it's possible to manually create it?

Reply:
We'll have to manually create it. Right now, changelogs are only in releases descriptions, so it won't be hard at least